manager/integration/tests/common.py

```diff
@@ -173,11 +173,11 @@ def cleanup_volume(client, volume):
     :param volume: The volume to clean up.
     """
     volume.detach()
-    volume = wait_for_volume_detached(client, volume["name"])
+    volume = wait_for_volume_detached(client, volume.name)
     client.delete(volume)
-    wait_for_volume_delete(client, volume["name"])
+    wait_for_volume_delete(client, volume.name)
     volumes = client.list_volume()
-    assert len(volumes) == 0
+    assert volumes.__len__() == 0
```

Comment:
Can you check if this PR fixes the inconsistency of __len__()? https://github.com/longhorn/longhorn-tests/pull/244

Reply:
ok, will check.

Reply:
PR works.